ukrainian/wait-more/wait.md

```diff
@@ -6,3 +6,4 @@
 Що будете робити далі?
 
 [Звичайно ж вийду з хатинки!](out/index.md)
+[Йти в дорогу не поснідавши? Це вже ні. Краще скуштую стіну з зефіру..](eat-wall/obese/diabetes.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.